### ISSUE TITLE
[BUG] Default icon shape 'System' not backed up

### ISSUE THREAD

**Issue by tgex0** [Deletescape-Media/Lawnchair#4732]
### Describe the bug

If you create a backup with the default icon shape 'System' (before changing the icon shape and changing it back), the 'System' icon shape is not backed up.

However, if you change the icon shape to anything other than 'System' and then change it back to 'System', it _will_ be backup up correctly.

### Steps to reproduce

1. With a fresh install, create a backup (you can change any other settings, just don't change the icon shape before making the backup... leave it at 'System')
2. Change the icon shape (to anything other than 'System')
3. Restore the backup
4. The icons remain at the shape chosen in Step 2 and do not revert to the 'System' shape which should be in the backup

### Expected behavior

I would expect the icon shape to always be backup (and restored correctly).

### Device information

- Device: POCO X5 5G
- OS: HyperOS 1.0.5.0 EEA (latest stock)
- App version: Lawnchair.Debug.14-dev.Nightly-CI_1294-74be2f9.apk

**Comment by SuperDragonXD** [comment]
Hello. This seems to be fixed in Lawnchair 15; as such, I'm closing this issue.

**Comment by tgex0** [comment]
Yes, indeed. I've just checked and this issues is fixed in Lawnchair 15.
